tools/coredump/testdata/amd64/ruby-3.3.9-loop.json

```diff
@@ -0,0 +1,87 @@
+{
+  "coredump-ref": "b30adc830b2f54cb27f8f35a1ac20c7c5b4fb6a81c9c272d62368f5194b3162e",
+  "threads": [
+    {
+      "lwp": 7397,
+      "frames": [
+        "is_prime+0 in /app/loop.rb:10",
+        "is_prime+0 in /app/loop.rb:10",
+        "sum_of_primes+0 in /app/loop.rb:20",
+        "<main>+0 in /app/loop.rb:30",
+        "libruby.so.3.3.9+0x31ba80",
+        "libruby.so.3.3.9+0x321007",
+        "libruby.so.3.3.9+0x22912d",
+        "libruby.so.3.3.9+0x303ada",
+        "libruby.so.3.3.9+0x30a691",
+        "libruby.so.3.3.9+0x317d3f",
+        "<main>+0 in /app/loop.rb:29",
+        "ffffffffffffffffffffffffffffffff+0x0",
```

Comment:
This and other similar spots inidicate that the symbolization is not fully working. Do you have any idea what causes this and the other erroneous frames?

Reply:
Perhaps someone who is an expert on those sections of the code can contribute that then.